lib/utils.pm

```diff
@@ -500,12 +500,20 @@ sub zypper_call {
     $IN_ZYPPER_CALL = 1;
     # Retrying workarounds
     my $ret;
+    my $search_conflicts = 'awk \'BEGIN {print "Processing Record - ",NR; group=0}
```

Comment:
@rwx788 @SergioAtSUSE @okurz I can't recall if we have code that will already detect such package conflicts (at least I could not find any)